Documentation/contributing.rst

```diff
@@ -181,6 +181,23 @@ If for some reason, running of the provisioning script fails, you should bring t
 
     $ vagrant halt
 
+Packer-CI-Build
+^^^^^^^^^^^^^^^^
+
+In cilium, we use a custom base box with a bunch of pre-installed libraries and
+tools that we need to our daily workflow. That base box is built with `Packer
+<https://www.packer.io/>`_ and it's hosted in the `packer-ci-build
+<https://github.com/cilium/packer-ci-build>`_ project.
+
+This base box can be updated and publish from `Jenkins
+<https://jenkins.cilium.io/job/cilium/job/packer-ci-build/job/master/>`_, where
+a new rebuild will update the `Vagrantcloud <https://app.vagrantup.com/cilium>`_
+and the `BUILD_ID <https://qa.nuxeo.org/jenkins/pipeline-syntax/globals#env>`_
+will be the version.  That version ID will be used in Cilium `Vagrantfiles
+<https://github.com/cilium/cilium/blob/master/test/Vagrantfile#L10>`_.
+
+In case that we want to make changes in packer-ci-build project, each PR in
```

Comment:
Change to the following:

Changes to this image are made via contributions to the `packer-ci-build` repository. Authorized GitHub users can trigger builds with a GitHub comment on the PR containing the trigger phrase ``build-me-please`` .